pkg/mock/keystore/mock_keystore.go

```diff
@@ -6,31 +6,76 @@ SPDX-License-Identifier: Apache-2.0
 
 package keystore
 
-import "sync"
+import (
+	"sync"
+
+	"github.com/trustbloc/edge-core/pkg/storage"
+)
 
 // MockProvider mocks keystore provider.
 type MockProvider struct {
-	Store     map[string][]byte
-	lock      sync.RWMutex
+	Store     *MockStore
```

Comment:
Can't we use the mock provider/store from [edge-core](https://github.com/trustbloc/edge-core/tree/master/pkg/storage/mockstore) ?